src/Microsoft.ML.Experimental/DataLoadSave/Database/DatabaseLoader.cs

```diff
@@ -490,6 +535,7 @@ public Bindings(ModelLoadContext ctx, DatabaseLoader parent)
                 //     ulong: count for key range
                 //   int: number of segments
                 //   foreach segment:
+                //     int: name
```

Comment:
`int` => `string`

Reply:
I think it should be `int: id of name`, to match what it is doing above: https://github.com/dotnet/machinelearning/pull/4308/files/90316ec3518890eeeaedab10f6ed1ad1b2b78853#diff-31c601a2df378d0ae48fc3007c11c5edR531

Reply:
I've changed to `string id: name`